iOS_SDK/OneSignalSDK/OneSignal.xcodeproj/project.pbxproj

```diff
@@ -1455,7 +1479,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CLANG_ANALYZER_NONNULL = YES;
-				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_MODULES = NO;
```

Comment:
Keep this as `YES` as it might be needed for Swift projects that consume the OneSignal SDK. `CLANG_ENABLE_MODULES`